cg/test/management/users.api.spec.js

```diff
@@ -154,3 +154,70 @@ describe('User password update', () => {
     expect((await res2.json()).jwt).toBeTruthy();
   });
 });
+
+describe('Listing management users', () => {
+  it('Should display an empty list when no manager is registered', async () => {
+    //Given
+    await db('users').del(); // It's fine to delete the contents of the managers table because no other table references it
```

Comment:
it will be ok, because of the order Jest runs tests.
Nice gist: https://gist.github.com/tamlyn/dd6d54cdbe6eccd51a4bc61f5844bec4

Reply:
We also could delete the 'users' table beforeEach test. It's good to have a clean state for each individual test

Reply:
it adds a lot of overhead, we should be able to get away with 1 db per file